src/mplfinance/_arg_validators.py

```diff
@@ -57,6 +57,16 @@ def _check_and_prepare_data(data, config):
 
     return dates, opens, highs, lows, closes, volumes
 
+def _label_validator(label_value):
+    if isinstance(label_value,str):
+        return True
+    elif not isinstance(label_value,tuple) and not isinstance(label_value,list):
```

Comment:
This can be simplified:
```python
elif not isinstance(label_value,tuple) and not isinstance(label_value,list):
```
becomes
```python
elif not isinstance(label_value,(tuple,list)):
```